README.md

```diff
@@ -14,20 +40,60 @@ IRC channel: [freenode](https://freenode.net) #googlebenchmark
 
 [Assembly Testing Documentation](docs/AssemblyTests.md)
 
+## Requirements
+
+The library can be used with C++03. However, it requires C++11 to build,
+including compiler and standard library support.
 
-## Building
+The following minimum versions are strongly recommended to build the library:
 
-The basic steps for configuring and building the library look like this:
+* GCC 4.8
+* Clang 3.4
+* Visual Studio 2013
+* Intel 2015 Update 1
+
+## Installation
+
+This describes the installation process using cmake. As pre-requisites, you'll
+need git and cmake installed.
 
 ```bash
+# Check out the library.
 $ git clone https://github.com/google/benchmark.git
 # Benchmark requires Google Test as a dependency. Add the source tree as a subdirectory.
 $ git clone https://github.com/google/googletest.git benchmark/googletest
+# Make a build directory to place the build output.
 $ mkdir build && cd build
-$ cmake -G <generator> [options] ../benchmark
-# Assuming a makefile generator was used
+# Generate a Makefile with cmake.
+# Use cmake -G <generator> to generate a different file type.
+$ cmake ../benchmark
+# Build the library.
 $ make
 ```
+This builds the `libbenchmark` and `libbenchmark_main` libraries and tests.
```

Comment:
Same here, `lib` prefix does not make much sense.

Reply:
Will update.